### PR TITLE
Limit the scheduler to a burst qps of 30

### PR DESCRIPTION
**Pull request by bprashanth**
@lavalamp @wojtek-t

### CODE REVIEW

**Comment by lavalamp** [comment]
@wojtek-t can you/we manually run this on the perf cluster?

**Comment by lavalamp** [comment]
LGTM, can't see any harm in merging...

**Comment by wojtek-t** [comment]
cc @fgrzadkowski 

>
@wojtek-t can you/we manually run this on the perf cluster?

Sure - either me or Filip will run it tomorrow.

**Comment by bprashanth** [comment]
Dont think tying the whole scheduler down to the binding qps is a good thing in the long run, so the new patch has a second way to rate limit just the bindings. Can choose whichever.

**Comment by wojtek-t** [comment]
@bprashanth - I think that in the offline discussion we agreed with @lavalamp that it's not fully fixing the introduced regression. So I think that we should revert the original PR #8261 and drop this one (at least short term).

**Comment by bprashanth** [comment]
Ok, though in theory this can exist without that since we can still create > 15 pods per second

**Comment by wojtek-t** [comment]
>
Ok, though in theory this can exist without that since we can still create > 15 pods per second

That's a good point - I will take a look later today.

File: plugin/pkg/scheduler/scheduler_test.go [hunk 1]
```diff
@@ -300,3 +300,70 @@ func TestSchedulerForgetAssumedPodAfterDelete(t *testing.T) {
...
+}
+
+func (fr *FakeRateLimiter) Stop() {}
+
+func (fr *FakeRateLimiter) Accept() {
+	fr.acceptValues = append(fr.acceptValues, fr.r.CanAccept())
+}
+
+func TestSchedulerRateLimitsBinding(t *testing.T) {
+	eventBroadcaster := record.NewBroadcaster()
```
Comment (wojtek-t) [inline plugin/pkg/scheduler/scheduler_test.go]: this doesn't seem to be used anywhere.
Is this really needed?

**Comment by wojtek-t** [comment]
LGTM - I agree it makes sense to have it submitted even without #8261

**Comment by bprashanth** [comment]
Addressed nit. This is certainly not a 1.0 blocker. Rate limits are just nice to have and relatively safe to put in.

File: plugin/pkg/scheduler/factory/factory.go [hunk 2]
```diff
@@ -38,6 +38,13 @@ import (
 	"github.com/golang/glog"
 )
 
+// Rate limitations for binding pods to hosts.
+// TODO: expose these are cmd line flags.
```
Comment (wojtek-t) [inline plugin/pkg/scheduler/factory/factory.go]: s/are/as/

**Comment by wojtek-t** [comment]
LGTM - just one small nit

**Comment by lavalamp** [comment]
Cool-- maybe we can try reverting #8533 after this goes in and see what happens.

**Comment by thockin** [comment]
CI failed

**Comment by bprashanth** [comment]
`TestNoManualChangesToGenerateDeepCopies` failed, which is a roundabout way of telling me I need a rebase

**Pull request merged**